raiden/network/rpc/client.py

```diff
@@ -337,6 +337,7 @@ def deploy_solidity_contract(
 
                 transaction_hash = self.send_transaction(
                     to=Address(b''),
+                    startgas=int(self.gaslimit() * 0.8),
```

Comment:
`gaslimit` already has the 0.8 factor, do you want to include it again (effectively making it 0.64 * last block gas limit)

Reply:
oops, nope. will fix